enhancements/network/user-defined-network-segmentation.md

```diff
@@ -413,6 +511,98 @@ guarantee uniqueness and eliminates the ability to falsify access.
 After creating the CRD, check the status to ensure OVN-Kubernetes has accepted this network to serve the namespaces
 selected. Now tenants may go ahead and be provisioned their namespace.
 
+#### NetworkAttachmentDefinition rendering
+
+The NAD should be created with owner-reference to the CRD, using the owner-reference mechanism should prevent deletion of the NAD before the corresponding CRD instance is deleted.
+Avoiding a state where the CRD instance exist but no corresponding NAD is.
+In addition, the owner-reference make teardown seamless, when the CRD instance is deleted, the cluster garbage collected will dispose the corresponding NAD.
+
+The underlying OVN network name, specified by the CNI conf `name` field, should be unique.
+Having the network name unique and non-configurable, mitigates the risk where a malicious entity with access to namespace "A" workloads could tap into network on namespace "B" by guessing its name.
+
+Creating namespace scoped CRD instance should trigger creation of a corresponding NAD at the namespace the CRD instance reside.
+Following the above [example](#example---namespace-scoped-network) above, the following NAD should be created:
```

Comment:
Do we need to add primary networks verifications like pods exists before creating a primary network ? 

Also check if an admin has create a primary network is already configured for the namespace.

Reply:
I will add these to validation section, thanks.